app/controllers/foreman_inventory_upload/tasks_controller.rb

```diff
@@ -0,0 +1,23 @@
+module ForemanInventoryUpload
+  class TasksController < ::ApplicationController
+    def create
+      selected_org = Organization.current
```

Comment:
`Organization.current` can be set to nil, if the user is not in organization's scope.